src/Core/Models/SystemUserRequestDTO.cs

```diff
@@ -13,7 +13,7 @@ namespace Altinn.Platform.Authentication.Core.Models
     public class SystemUserRequestDto
     {
         [AllowNull]
-        [JsonPropertyName("PartyId")]
+        [JsonPropertyName("partyId")]
```

Comment:
i remember we had a discussion about saving both partyid and organization number of the org that creates the system user. I don't see the field addition and property for the same here

Reply:
That is right, will add it.